YunoHost/yunohost
# [muc subdomain] add to domain's certificate the alt subdomain muc

## The problem

In metronome SASL on multi user chat always failed and fallback on dialback method.
For some servers (eg. 404.city) this is not acceptable and they refuse to connect the user to the room.

## Solution

Add muc subdomain as subjectAltName in the letsencrypt certificate.

## PR Status

Need to be approved. 

## How to test

Create a room hosted on yunohost server, with participants from other servers.
- You should see in metronome log file (/var/log/metronome/metronome.log) "muc.{domain}.*SASL.*fail"
and also "dialback"
- You shouldn't be able to add in the conversation a user from 404.city

Apply the change, regenerate your domain certificate.
- You should be able to add a user from 404.city
